src/test/java/com/powsybl/openloadflow/ac/MultipleSlackBusesTest.java

```diff
@@ -148,20 +157,26 @@ void nonImpedantBranchTest(boolean ac) {
         assertSlackBusResults(slackBusResults, expectedSlackBusMismatch, 2);
     }
 
-    @ParameterizedTest(name = "ac : {0}")
-    @MethodSource("allModelTypes")
-    void loadOnSlackBusTest(boolean ac) {
+    @ParameterizedTest(name = "ac : {0}, switchSlacks : {1}")
+    @MethodSource("allModelAndSwitchingTypes")
+    void loadOnSlackBusTest(boolean ac, boolean switchSlacks) {
```

Comment:
Could you add another test that runs one (or more models) with one slack and without multiple and checks that the solution is close enough ? (bus injections, bus voltage, terminal flows...)

Reply:
Yes, done a few minutes ago, I'll add more asserts to check bus injections, bus voltages as you mentioned

Reply:
Done